src/Pyramid-Bloc/PyramidMainExtension.class.st

```diff
@@ -143,24 +142,20 @@ PyramidMainExtension >> offsetTranslation [
 	^ 50 @ 50
 ]
 
-{ #category : 'accessing' }
-PyramidMainExtension >> projectModel [
-
-	^ projectModel
-]
-
 { #category : 'accessing' }
 PyramidMainExtension >> projectModel: aProjectModel [
 
-	projectModel:= aProjectModel.
-	aProjectModel addObserver: self.
+	aProjectModel announcer
+		when: PyramidFirstLevelElementsChangedEvent
+		send: #pyramidFirstLevelElementsChanged:
```

Comment:
Ok. But does use a symbol here for "send:" is the right way? How it is done in Bloc?